scripts/integration_tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
I would recommend not to have this kind of script, separate JS and RS jobs on the CI and just using ava (as a yarn test in package.json) and cargo to run them.